src/slipcover/slipcover.py

```diff
@@ -1,23 +1,27 @@
 from __future__ import annotations
+import functools
 import sys
 import dis
 import types
-from typing import Dict, Set, List, Tuple, Optional, Iterator, cast
+from typing import Dict, Set, List, Tuple, TYPE_CHECKING, Iterator
 from collections import defaultdict, Counter
 import threading
 
-if sys.version_info < (3,12):
-    from . import probe # type: ignore[attr-defined]
+if sys.version_info[0:2] < (3,12):
```

Comment:
We don't need the `[0:2]`, and I'm pretty sure I've seen versions of MyPy not work correctly with the slice...
Please revert these changes.

Reply:
i...have no idea how this happened sorry 😅! I swear I thought it was already like this ^ and I was just matching the existing pattern...

I've only ever used the original pattern in my life lol

Reply:
Perhaps!